pegasus/sites.v3/hourofcode.com/views/home_signup.haml

```diff
@@ -4,9 +4,7 @@
 %a#hocsignupform{name: 'signup'}
 #join-us-header=hoc_s(:front_join_us_button)
 - if ["actual-hoc", "post-hoc", false].include?(hoc_mode)
-  #signup-closed
-    !=hoc_s(:signup_registration_not_required_markdown, markdown: :inline, locals: {hoc_activities_url: resolve_url('/learn'), howto_guide_url: resolve_url('/how-to')})
-    =hoc_s(:signup_registration_closed)
+  %h4=hoc_s(:signup_registration_closed_2022_markdown, markdown: :inline, locals: {hoc_activities_url: resolve_url('/learn')})
```

Comment:
Should this be `h4`?  It feels like bold body content more than a heading.  If we are doing this just to inherit a bit of `h4` styling, then maybe we could just duplicate that with a new CSS class.

Reply:
I used an `h4` since this messaging feels higher priority than a `p`, like more of an alert or announcement, but happy to change it to a `p` if you think it's necessary!

Reply:
I don't have super strong feelings; but I think it's probably a good overall practice to reserve the `hX` tags for what are truly headings.